README.md

```diff
@@ -89,203 +59,101 @@ You can specify the project files in `polymer.json` so that commands like `polym
-      }
-    }
-  }
-}
-```
-
-When using `polymer install --variants`, this results in two additional folders,
-`bower_components-polymer-1` and `bower_components-polymer-2`, installed as if
-they had their own `bower.json` files as follows:
-
-`bower_components-polymer-1`:
-
-```json
-{
-  "name": "example",
-  "dependencies": {
-    "polymer": "Polymer/polymer#^1.7.0",
-    "foobar": "Foo/bar#^2.0.0"
-  }
-}
-```
-
-`bower_components-polymer-2`:
 
-```json
-{
-  "name": "example",
-  "dependencies": {
-    "polymer": "Polymer/polymer#^2.0.0",
-    "foobar": "Foo/bar#^2.0.0"
-  }
-}
-```
-
-Commands like `install`, `serve` and `test` recognize that variants are
-installed to offer additional support.
-
-`polymer install --variants` installs dependency variants into the proper
-folders.
+## Command Overview
 
-`polymer serve` will start up a development server for
-each variant on its own port so that you can easily switch between variants,
-viewing their demos and tests.
+### `polymer help [COMMAND]`
 
-`polymer test` will run the test suite once for each variant, ensuring that you
-have test coverage against supported configurations.
+Run `polymer help` to get a helpful list of supported commands. Pass it a command name (ex: `polymer help serve`) to get detailed information about that command and the options it supports.
 
-## Commands
 
-### help
+### `polymer init [TEMPLATE]`
 
-Displays help on commands and options:
+Initializes a Polymer project from one of several templates. Pre-bundled templates range from just bare-bones to fully featured applications like the [Polymer Starter Kit](https://github.com/PolymerElements/polymer-starter-kit).
 
-    $ polymer help
+You can download and run templates built by our community as well. [Search npm](https://www.npmjs.com/search?q=generator-polymer-init) for a template you'd like to use. Then install it and the CLI will pick it up automatically.
 
-### init
+Run `polymer init` to choose a template from a list of all installed templates. Or if you know the template name before hand, you can provide it as a command argument.
 
-Initializes a Polymer project from one of several templates.
 
-Choose a template from a menu:
+### `polymer install [--variants]`
 
-    $ polymer init
+Install your dependencies, similar to running `bower install`.
 
-Create a new project from the 'element' template:
+If the `--variants` option is provided, the command will also search your project's `bower.json` for  a `"variants"` property and install any [dependency variants](https://www.polymer-project.org/2.0/docs/glossary#dependency-variants) listed there as well as well. This is useful if you need to test your elements against multiple versions of Polymer and/or sets of dependencies.
 
-    $ polymer init element
 
-You can download and run templates built by our community as well. [Search npm](https://www.npmjs.com/search?q=generator-polymer-init) for the full list.
+### `polymer serve [options...]`
```

Comment:
Yes, logically it's not required but it helps to point out that there are significant features hidden behind the commands where it's included. For example, I think this lets us get around having to go into detail to explain `--open` and others